consent-ui/src/app/pis/to-aspsp-page/to-aspsp-page.component.html

```diff
@@ -0,0 +1,8 @@
+<consent-app-to-aspsp
+  [aspspName]="aspspName"
+  [finTechName]="finTechName"
+  [actionName]="'payment'"
```

Comment:
Quotes `'` make this to be string constant, not enum

Reply:
Oh yes of course